File: api/code-samples/python/klay/block/getBlockByNumber.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0x1b4"
boolean = True

sdk = OpenSDK(host)
klay_response = sdk.klay.get_block_by_number(blockTag, boolean)

print(klay_response)

File: api/code-samples/javascript/klay/block/getBlockByNumber.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockNumber = 1
    const returnTransactionObject = true
    
    sdk.klay.getBlockByNumber(blockNumber, returnTransactionObject, {}, (err, data, response) => {
        console.log(data);
    });
}
)()